lib/src/typings/helpers.ts

```diff
@@ -1,5 +1,3 @@
-import { Omit } from '../_helpers/utils';
```

Comment:
I see that the helper was introduced in the TypeScript (v3.5) directly: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-5.html. However, I'm wondering about what version of TypeScript we should support and the timing. cc @eps1lon. (TypeScript doesn't follow semver).

On a side note, we have this helper in `@material-ui/types` too. https://unpkg.com/browse/@material-ui/types@4.1.1/index.d.ts

Reply:
We are already using `Omit` from typescript in v3. It was accidentally a problem, so in the release notes we asked to update typescript